omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -273,6 +273,42 @@
 default['private_chef']['opscode-erchef']['sql_password'] = "snakepliskin"
 default['private_chef']['opscode-erchef']['sql_ro_user'] = "opscode_chef_ro"
 default['private_chef']['opscode-erchef']['sql_ro_password'] = "shmunzeltazzen"
+#
+# Reindex configurables
+#
+# NOTE: The semantics of reindexing are very different between
+# ElasticSearch and Solr. Solr reindexing is only placing an item on a
+# queue, thus it is unlikely to fail at the erchef level.  Rather, you
+# will want to tune opscode-expander.
+#
+# These configuration items are consulted during reindex requests.
+# Such requests originate from users running chef-server-ctl reindex.
+#
+#   reindex_batch_size - Number of items to fetch from the database
```

Comment:
I think that depends on whether you think the configurable names should be left or right justified ;)